### PR TITLE
Adds tags as an optional package field.

### PR DESCRIPTION
**Pull request by ConnorDoyle**

### CODE REVIEW

File: repo/meta/schema/package-schema.json [hunk 1]
```diff
@@ -65,6 +65,13 @@
           "version"
         ]
       }
+    },
+    "tags": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "[a-z]+"
```
Comment (jsancio) [inline repo/meta/schema/package-schema.json]: why the restriction of only lower case letters? For example users want to tag something as `python3`.

Comment (ConnorDoyle) [inline repo/meta/schema/package-schema.json]: Good call, will open this up to allow numbers as well.

Comment (ConnorDoyle) [inline repo/meta/schema/package-schema.json]: To be even less restrictive, we'll allow everything except whitespace here.

Comment (ConnorDoyle) [inline repo/meta/schema/package-schema.json]: Done, just needs to be a nonempty string without whitespace.

**Pull request merged**